content/articles/searching-and-sorting-in-matlab/index.md

```diff
@@ -0,0 +1,89 @@
+### SEARCHING AND SORTING USING MATLAB
+### Introduction
+Searching is the act of going or looking through (a place, area, etc.) carefully in order to find something while sorting Sorting is any process of arranging items systematically. This is something that is mostly popular in databases.
+Here, we will look at the two closely related problems: Searching and sorting. Both problems appear in applications involving databases, and without efficient solutions to these problems, databases would be virtually useless. They would be useless because databases that are large enough to hold the information we need would be too large to use. They would be too large to use because it would take too long to find what we are looking for inside them.
+In this article we will look at the how to apply the Searching and sorting in Matlab.
+
+### Prerequisites
+- [Matlab](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- Proper understanding of [matlab](https://www.section.io/engineering-education/getting-started-with-matlab/) basics.
+
+
```

Comment:
After prerequisites, kindly connect the article, maybe use a subheading and build on what you had introduced.
Make the article flowing